project/accounts/templates/accounts/user_civis.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
```

Comment:
Load i18n for translation support.

```suggestion
{% load i18n %}
```